packages/venia-ui/.storybook/getEnvironmentVariable.js

```diff
@@ -0,0 +1,24 @@
+const envVarDefinitions = require('@magento/pwa-buildpack/envVarDefinitions.json');
+
+/**
+ * Returns the value of the first variable in any section whose name matches the input.
+ * @param {string} targetVariable - The name of the environment variable to find.
+ */
+const getEnvironmentVariable = targetVariable => {
+    let match;
+    envVarDefinitions.sections.forEach(section => {
+        section.variables.forEach(variable => {
+            if (variable.name === targetVariable) {
+                match = variable;
```

Comment:
What if there are multiple variables with `name === targetVariable`?

If there is only 1 that is expected or only the first one that is needed, consider using a `for of` loop so you can `break` the loop once you are in the `if` case.